examples/motorbike-application/jobbergate.py

```diff
@@ -1,5 +1,5 @@
 from jobbergate_cli.subapps.applications.application_base import JobbergateApplicationBase
-from jobbergate_cli.subapps.applications.questions import Text, Integer
+from jobbergate_cli.subapps.applications.questions import Text, Integer, Directory
```

Comment:
you're not actually using this new import, are you?

Reply:
resolved in latest commit.